phaser-react/src/states/flow1/Death.jsx

```diff
@@ -3,7 +3,8 @@
 export default function Death({ send }) {
   return (
     <div>
-        <h1>It's been a lonng journey through the desert. You ran out of water. You have died from heat exhaustion.</h1>
+        <h1>Death</h1>
+          <p>It&#39;s been a long journey through the desert. You ran out of water. You have died from heat exhaustion.</p>
```

Comment:
I'm not sure if you meant to write `It&#39;s` instead of `It's`, but I wanted to flag it just in case.

Reply:
It's addressing an ESlint rule to prevent escaped characters from being rendered. Thanks for being so meticulous in your review though and flagging it. I appreciate it!
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md